modules/selection/main/index.coffee

```diff
@@ -381,7 +381,7 @@ class Selection
 
       data.listenerNamesRegistered ?= new hx.Set
 
-      if not data.listenerNamesRegistered.has(name)
+      if name.indexOf('pointer') is -1 and not data.listenerNamesRegistered.has(name)
```

Comment:
Would `name.indexOf('pointer') isnt 0` suffice? This could catch any events with pointer in the name (if there are any others)